docs/pbc/all/service-point-management/202311.0/unified-commerce/service-points-feature-overview.md

```diff
@@ -64,6 +65,35 @@ After placing an order, the customer can see the selected service point on the O
 
 ![Storefront order with a service point](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/service-point-management/unified-commerce/service-points-feature-overview.md/storefront-order-service-point.png)
 
+
+## Current constraints
+
+* Currently, only product offers are supported, which means that services can be configured exclusively at the level of a product offer
+* Product Catalog is not supported, meaning it can not be filtered by the service type or service provided in the specific service point
+* Product details page is not supported, meaning it will show all product offers created in the system for the product
+* Product can not be added to the shopping cart with pre-selected service point, this can be done only during checkout process
+* If a product was added to the shopping cart without a product offer attached to it then this product can not be purchased with a shipment type other than *Delivery*.
+
+
+Similar  constraints apply to the *shipment type*. See [Shipment feature overview](/docs/pbc/all/carrier-management/{{page.version}}/base-shop/shipment-feature-overview.html).
+
+## Recommended customizations for Click & Collect setup
+
+Since we do not yet support a product catalog or a product details page, the following project-level customizations are recommended on the Product Details Page:
+
+* First, you need to implement the Default Product Offer Reference Strategy if you want a specific offer to be prioritized at the first position. Refer to the [Install the Marketplace Product Offer features](https://docs.spryker.com/docs/pbc/all/offer-management/202307.0/marketplace/install-and-upgrade/install-features/install-the-marketplace-product-offer-feature.html#set-up-behavior)  development documentation for more details. Ensure that only product offers are displayed on prodcut details page. 
+* Second, you need to filter the list of product offers displayed on the Product Details page by overriding the Merchant Product Offer Widget behavior. This filter should display product offers with an empty service and shipment type set to _empty_ or _Delivery_. In this case, customers will always be able to buy a product with the default shipment option and, if desired, choose Pickup during checkout." Refer to the [](https://docs.spryker.com/docs/pbc/all/offer-management/202307.0/marketplace/render-merchant-product-offers-on-the-storefront.html#prerequisites).
+* Add information about product availability in the service point locations by using Service Point Widget. Refer to [Install the Service Points feature](https://docs.spryker.com/docs/pbc/all/service-point-management/202311.0/unified-commerce/install-features/install-the-service-points-feature.html) development documentation for more details.
```

Comment:
In the Install the Service Points feature, the installation of the widget is not optional. So why are we referring to this as a customization when it comes as part of the feautre?

Reply:
The widget OOTB displayed only on checkout, but on PDP you need to add some code to call for it, this is just a suggested / recommended way to build an end-to-end UX. We could not do it OOTB cos we keep vanilla code base clean from specific implementations. Customer can choose to go with more sofisticated approuch if they want to. We just saying this is a easiest one with almost no effrots, other UX require more changes on the project level.